app/code/Magento/Sales/Model/CronJob/CleanExpiredOrders.php

```diff
@@ -16,20 +20,28 @@ class CleanExpiredOrders
     protected $storesConfig;
 
     /**
-     * @var \Magento\Sales\Model\ResourceModel\Order\CollectionFactory
+     * @var CollectionFactory
      */
     protected $orderCollectionFactory;
 
+    /**
+     * @var OrderManagementInterface
+     */
+    private $orderManagement;
+
     /**
      * @param StoresConfig $storesConfig
-     * @param \Magento\Sales\Model\ResourceModel\Order\CollectionFactory $collectionFactory
+     * @param CollectionFactory $collectionFactory
+     * @param OrderManagementInterface $orderManagement
      */
     public function __construct(
         StoresConfig $storesConfig,
-        \Magento\Sales\Model\ResourceModel\Order\CollectionFactory $collectionFactory
+        CollectionFactory $collectionFactory,
+        OrderManagementInterface $orderManagement
```

Comment:
Hi, @JeroenVanLeusden, due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add a new required parameter to the constructor.

Reply:
Of course was in a rush, give me couple of minutes to fix this.